listenbrainz/db/color.py

```diff
@@ -127,3 +127,35 @@ def get_releases_for_color(red: int, green: int, blue: int, count: int) -> List[
                     results[i].rec_metadata = recordings
 
         return results
+
+
+def fetch_color_for_releases(release_mbids: List[str]) -> Dict[str, Dict[str, int]]:
```

Comment:
You can accept the cursor here as method argument instead of creating a new connection.

Reply:
I don't think its needed here, because we're not using the LB connection in places where this function is being called